alert.go

```diff
@@ -20,7 +20,7 @@ import (
 	imodels "github.com/influxdata/influxdb/models"
 	"github.com/influxdata/kapacitor/expvar"
 	"github.com/influxdata/kapacitor/models"
-	"github.com/influxdata/kapacitor/pipeline"
+	"./pipeline"
```

Comment:
This change is incorrect, you can leave it how it was.

This may be an indication that your GOPATH is not setup correctly yet.